app/services/onlinetesting/OnlineTestService.scala

```diff
@@ -183,20 +181,19 @@ trait OnlineTestService {
   }
 
   private def emailInviteToApplicant(application: OnlineTestApplication, emailAddress: String,
-    invitationDate: DateTime, expirationDate: DateTime): Future[Unit] = {
+                                     invitationDate: DateTime, expirationDate: DateTime): Future[Unit] = {
     val preferredName = application.preferredName
     emailClient.sendOnlineTestInvitation(emailAddress, preferredName, expirationDate).map { _ =>
       audit("OnlineTestInvitationEmailSent", application.userId, Some(emailAddress))
     }
   }
 
   private def markAsInvited(application: OnlineTestApplication)
-    (onlineTestProfile: Phase1TestProfile): Future[Unit] = for {
-    _ <- otRepository.insertPhase1TestProfile(application.applicationId, onlineTestProfile)
-    _ <- appRepository.updateProgressStatus(application.applicationId, ProgressStatuses.PHASE1_TESTS_INVITED)
+                           (onlineTestProfile: Phase1TestProfile): Future[Unit] = for {
+    _ <- appRepository.insertPhase1TestProfile(application.applicationId, onlineTestProfile)
   } yield {
-      audit(s"ApplicationStatus set to ${ApplicationStatus.PHASE1_TESTS} - ProgressStatus set to" +
-        s" ${ProgressStatuses.PHASE1_TESTS_INVITED}", application.userId)
+    audit(s"ApplicationStatus set to ${ApplicationStatus.PHASE1_TESTS} - ProgressStatus set to" +
```

Comment:
The code is from the other branch (merge) - I'll update this here, but we need to be careful when merging later to not override the fix